### PR TITLE
message definitions for time series data points

### PR DESCRIPTION
**Pull request by larshagencognite**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM. (Maybe add empty line at the end of each file?)

File: v1/timeseries/data_point_insertion_request.proto [hunk 1]
```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package v1.timeseries;
```
Comment (martincognite) [inline v1/timeseries/data_point_insertion_request.proto]: `com.cognite.v1.timeseries`

What about following the standard naming convention? We're doing this most other places as far as I know (at least seismic and opint).

File: v1/timeseries/data_point_insertion_request.proto [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import "data_points.proto";
+
+option java_package = "com.cognite.api.v1.timeseries.proto";
+option csharp_namespace = "Cognite.Api.V1.TimeSeries.Proto";
+option java_multiple_files = true;
```
Comment (simenl) [inline v1/timeseries/data_point_insertion_request.proto:7]: Do we need to define the options here? Or should it be handled by the user?

File: v1/timeseries/data_point_list_response.proto [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+option java_package = "com.cognite.api.v1.timeseries.proto";
+option csharp_namespace = "Cognite.Api.V1.TimeSeries.Proto";
+option java_multiple_files = true;
+
+message DataPointListItem {
+    int64 id = 1;
+    string externalId = 2;
+
+    oneof numericOrStringDatapoints {
```
Comment (simenl) [inline v1/timeseries/data_point_list_response.proto]: -> datapointsType ?
As you also have AggregateDatapoints

**Review by simenl — APPROVED** [review]

File: v1/timeseries/data_point_insertion_request.proto (hunk 1 above)
Comment (larshagencognite) [reply v1/timeseries/data_point_insertion_request.proto]: I added .proto to the package to make it not clash with internal classes in our code so easily

**Pull request merged**